docs/pages/deployment/storage.rst

```diff
@@ -60,7 +60,6 @@ Microsoft Azure Key Vault
 This storage backend uses Microsoft Azure's Key Vault. It authenticates to the Azure Key Vault at the configured URL using the default credential,
 typically an Azure Managed Identity. Refer to the `Azure SDK for Go documentation <https://github.com/Azure/azure-sdk-for-go/wiki/Set-up-Your-Environment-for-Authentication>`_ for more information.
 
-- Azure Key Vault storage can't be used for nodes that produce ``did:nuts`` DIDs or for data encryption.
```

Comment:
I think this is still true?

Reply:
no there's a SQL table in between now

Reply:
but that doesn't change that did:nuts uses ecies to encrypt the `pal` header that is not supported by azure.
Or can the header also be encrypted using RSA?

Reply:
ahh yes, you can't use did:nuts with private credentials over the network